src/SFA.DAS.AssessorService.Database/Script.PostDeployment1.sql

```diff
@@ -90,5 +90,17 @@ END
 
 -- ON-2242
 :r .\Update-Staff-Reports-Config.sql
-
 -- END
+
+-- ON-2088
+IF NOT EXISTS(SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[dbo].[AppRefSequence]') AND type = 'SO')
+CREATE SEQUENCE [dbo].[AppRefSequence] 
+ AS [int]
+ START WITH 100001
```

Comment:
Also this is a new sequence in a different database which starts with the same sequence the original one did in the old database - is would need to be synchronised to start at the point the old one left off on migration

Reply:
I know this needs to start from the correct number when deployed on PP and Prod